src/miner.cpp

```diff
@@ -183,6 +186,9 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& s
 
             const CTransaction& tx = iter->GetTx();
 
+            if (!fIncludeWitness && !tx.wit.IsNull())
```

Comment:
Going to replace it with an extra delay before witness transactions are accepted to the mempool.